api/integrations/google/oauth.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import datetime
+import httplib2
+import os
+
+# Uses Google's Client Library
+#   pip install --upgrade google-api-python-client
+from oauth2client import client, tools
+from oauth2client.file import Storage
+
+parent = tools.argparser
```

Comment:
Need to add a comment to explain why you are using argument parser from `tools` instead of `argparse`.